app/views/councils/_form.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_for council, remote: true do |f| %>
+  <%= f.text_field :council_name,  placeholder: "Council Name"  %>
```

Comment:
Should this be remote :true? Are you using rails ajax:success etc events in the javascript? It's better to be consistent, leave this off, and use the standard approach (preventDefault and call $.ajax) Don't mix ajax approaches in one project.